modules/archive/functions/lsarchive

```diff
@@ -43,9 +43,11 @@ while (( $# > 0 )); do
       || lzcat "$1" | tar x${verbose:+v}f - ;;
     (*.tar) tar t${verbose:+v}f "$1" ;;
     (*.zip|*.jar) unzip -l${verbose:+v} "$1" ;;
-    (*.rar) unrar &> /dev/null \
+    (*.rar) unrar -V &> /dev/null \
```

Comment:
It would probably be better to use a check like `(( ${+commands[unrar]} ))` here and for the other commands.